src/ruuvi_endpoint_3.c

```diff
@@ -57,8 +57,7 @@ static void re_3_encode_acceleration (uint8_t * const buffer,
 {
     if (!isnan (re_3_encode_check_invalid (acceleration, invalid)))
     {
-        int16_t decimal = (int16_t) round ( (acceleration *
-                                             RE_3_ENCODE_ACC_CONVERT_RATIO));
+        int16_t decimal = (int16_t) roundf (acceleration * RE_3_ENCODE_ACC_CONVERT_RATIO);
```

Comment:
I think the original intention was to wrap the RE_3_ENCODE_ACC_CONVERT_RATIO inside braces, just in case it's a math operation.